styleguide/components/31-molecules/cta-block/cta-block--double.twig

```diff
@@ -0,0 +1,4 @@
+<div class="cta-block--multiple">
+  {% include 'cta-block.twig' with items[0]|merge({'top' : true}) %}
+  {% include 'cta-block.twig' with items[1]|merge({'top' : true}) %}
```

Comment:
Why not include the `top` property in `cta-block.config.js`?

Reply:
because it's not configurable